pkgs/development/libraries/ncurses/default.nix

```
@@ -19,7 +19,10 @@ stdenv.mkDerivation rec {
   pname = "ncurses" + lib.optionalString (abiVersion == "5") "-abi5-compat";
 
   src = fetchurl {
-    url = "https://invisible-island.net/archives/ncurses/current/ncurses-${ver}-${patchver}.tgz";
+    # https://github.com/ThomasDickey/ncurses-snapshots/archive/refs/tags/v6_3_20220507.tar.gz
+    # has the same source code, but the archive differs. Since we're using
+    # fetchurl, we use archive.org here, keeping the hash and avoiding rebuilds
```

```suggestion
    # old snapshots are removed
```
yeah, that can happen because those tarballs are generated by git archive.